apps/cookbook/src/app/showcase/list-showcase/list-showcase.component.html

```diff
@@ -12,22 +12,22 @@
 
 <div class="page-example">
   <div>
-    <h2>List with items</h2>
-    <cookbook-code-viewer [html]="items"></cookbook-code-viewer>
+    <h2>List with items using dividers (default):</h2>
+    <cookbook-code-viewer [html]="dividersExampleTemplate"></cookbook-code-viewer>
```

Comment:
I would prefer if we can rename the example components so `items` is the one with dividers, and the one used in the no dividers list will be called `NoDividersExampleComponent` or something like it.